src/test/java/redis/clients/jedis/modules/search/SearchTest.java

```diff
@@ -443,6 +438,42 @@ public void testQueryFlags() throws Exception {
     assertEquals(0, res.getTotalResults());
   }
 
+  @Test
+  public void testVectorSimilarity() {
```

Comment:
taken from one of the module's tests : https://github.com/RediSearch/RediSearch/blob/e139248127a17c81cddad840fca2fd99e80f5f41/tests/pytests/test_search_params.py#L312